packages/web/src/components/basic/ReactiveComponent.js

```diff
@@ -159,6 +159,19 @@ class ReactiveComponent extends Component {
 			);
 		}
 
+		checkPropChange(this.props.selectedValue, prevProps.selectedValue, () => {
+			/*
+				Reset query when SelectedFilters are clicked. Note: `selectedValue` becomes null.
+			*/
+
+			if (this.props.selectedValue === null) {
+				this.props.updateQuery({
```

Comment:
@jyash97 Instead of updating the query you should set the value to `null`. Use `setValue` method.
This code will not work when `enableAppbase` is `true`.

Reply:
I have tested with enableAppbase as well it works.

Reply:
Do check the video attached